app/src/client/endpoints.ts

```diff
@@ -125,6 +125,29 @@ export class GetCollectionStatisticsResponse extends ResponseData {
   }
 }
 
+export class GetCollectionRecentSalesRequest extends RequestData {
+  public toObject = (): Record<string, unknown> => {
+    return {
+    };
+  }
+}
+
+export class GetCollectionRecentSalesResponse extends ResponseData {
+  readonly collectionRecentSales: Resources.CollectionRecentSales;
```

Comment:
this should just be `tokenTransfers: TokenTransfer[]` no need for a new resource here it looks like